.github/actions/next-stats-action/src/index.js

```diff
@@ -108,22 +108,22 @@ if (!allowedActions.has(actionInfo.actionName) && !actionInfo.isRelease) {
       if (!actionInfo.skipClone) {
         const usePnpm = await fs.pathExists(path.join(dir, 'pnpm-lock.yaml'))
 
-        let buildCommand = `cd ${dir}${
-          !statsConfig.skipInitialInstall
```

Comment:
Do we no longer use `skipInitialInstall`?

Reply:
Maybe also remove it from the `README.md`

Reply:
Re-added, mostly was used for local testing but doesn't hurt to keep around